authentik/sources/ldap/migrations/0005_remove_ldappropertymapping_object_field_and_more.py

```diff
@@ -0,0 +1,71 @@
+# Generated by Django 5.0.2 on 2024-02-29 11:21
+
+import textwrap
+
+from django.db import migrations
+
+
+def migrate_ldap_property_mappings_object_field(apps, schema_editor):
+    if schema_editor.connection.alias != "default":
+        return
+    LDAPPropertyMapping = apps.get_model("authentik_sources_ldap", "LDAPPropertyMapping")
+    for mapping in LDAPPropertyMapping.objects.all():
+        mapping.expression = f"""
+# This property mapping has been automatically changed to
+# match the new semantics of source property mappings.
+# You can simplify it if you want.
+# You should return a dictionary of fields to set on the user or the group.
+# For instance:
+# return {{
+#     "{mapping.object_field}": ldap.get("{mapping.object_field}")
+# }}
```

Comment:
```python
# return {
#     "{mapping.object_field}": ldap.get("{mapping.object_field}")
# }
```

Reply:
We need that `{{` so they're not escaped by the f-string